tuna/example/load_job.py

```diff
@@ -53,19 +56,19 @@ def parse_args():
                       help='Label to annontate the jobs.',
                       default='new')
 
-  args = parser.parse_args()
+  args: argparse.Namespace = parser.parse_args()
   if not args.session_id:
     parser.error('session_id must be specified')
 
   return args
 
 
-def add_jobs(args, dbt):
+def add_jobs(args: argparse.Namespace, dbt: ExampleDBTables) -> int:
```

Comment:
It's seems to be outside the scope of this PR, but passing around `args` doesn't seem ideal. It's unclear what args.* the function needs in order to work. Had the signature been `def add_jobs(label: str, session_id: int, dbt: XYZ)`, it'd be clear outright what the function needs. Or, perhaps, one can create `TunaNamespace` as a subclass of argparse.Namespace, and pass that around? @cderb

Reply:
@alexandraBara

Reply:
@msaudulhassan fair enough, but should mark that out of scope. Perhaps @technicalgrp89 can add a comment to the function that this update should be made for clarity.

Reply:
I will add the comment